emission/net/api/cfc_webapp.py

```diff
@@ -388,32 +389,59 @@ def getCarbonCompare():
 @get('/eval/publicData/timeseries')
```

Comment:
Also, this is getting to be pretty big now. I think that you should move the implementation to a separate file/module, implemented in emission (e.g. in `emission/net/api/eval.py`?) The goal is have the bottle file be a thin shim over real functionality in the api classes. That way we can switch to a different frontend later.